src/auth/googleauth.ts

```diff
@@ -195,8 +206,13 @@ export class GoogleAuth<T extends AuthClient = JSONClient> {
 
   /**
    * Obtains the default project ID for the application.
-   * @param callback Optional callback
```

Comment:
nit: did you mean to delete the @param + @returns fields or just add after?

Reply:
Good question - I meant to delete as it could cause confusion (e.g. if using a promise, no need for callback param).